orca-retrofit/orca-retrofit.gradle

```diff
@@ -30,13 +30,15 @@ dependencies {
   implementation("io.reactivex:rxjava")
   implementation("io.spinnaker.kork:kork-retrofit")
   implementation("com.jakewharton.retrofit:retrofit1-okhttp3-client:1.1.0")
+  implementation("com.squareup.retrofit2:converter-jackson")
```

Comment:
it's part of the test setup [here](https://github.com/spinnaker/orca/blob/5a58ab932b7b59aab7da6e064b3e97e61faa1412/orca-retrofit/src/test/java/com/netflix/spinnaker/orca/retrofit/exceptions/SpinnakerServerExceptionHandlerTest.java#L166). I guess maybe it should be a `testImplementation` for now since it's currently only needed to generate the retrofit2 exceptions for testing.

Reply:
Yeah, it does mean a bit more juggling, but it does make things easier to follow.